libs/ui-lib/lib/ocm/components/clusterConfiguration/HostInventory.tsx

```diff
@@ -24,15 +24,11 @@ import { DiscoveryImageModalButton } from './DiscoveryImageModal';
 import InformationAndAlerts from './InformationAndAlerts';
 import { OcmSwitchField } from '../ui/OcmFormFields';
 import { selectCurrentClusterPermissionsState } from '../../selectors';
-import PlatformIntegration from './platformIntegration/PlatformIntegration';
 
 const schedulableMastersTooltip =
   'Workloads must be run on control plane nodes when less than 5 hosts are discovered';
 
 const HostInventory = ({ cluster }: { cluster: Cluster }) => {
-  const isPlatformIntegrationFeatureEnabled = useFeature(
-    'ASSISTED_INSTALLER_PLATFORM_INTEGRATION_FEATURE',
```

Comment:
`ASSISTED_INSTALLER_PLATFORM_INTEGRATION_FEATURE` from `featureGate` needs to be removed given it's not used anymore

Reply:
@jkilzi do we need this feature to show the dropdown to the  user?
In master we have this code: https://github.com/openshift-assisted/assisted-installer-ui/blob/master/libs/ui-lib/lib/ocm/components/clusterConfiguration/HostInventory.tsx#L64

Reply:
Then, do we need to remove this feature from the code?